src/global/mixins/mixins_typo.scss

```diff
@@ -142,3 +142,10 @@
     color: var(--color-metal-default);
   }
 }
+
+@mixin clampLinesByTwo() {
+  overflow: hidden;
+  display: -webkit-box;
+  -webkit-box-orient: vertical;
+  -webkit-line-clamp: 2;
```

Comment:
looks weird, but seems to work also in firefox. Did you test various browser for support?

Reply:
Yes, this works really fine and I tested it in ff, safari and chrome. Only in IE there is no support for that.